stdlib/public/core/Collection.swift

```diff
@@ -97,6 +99,8 @@ public protocol _IndexableBase {
   /// - Parameter bounds: A range of the collection's indices. The upper and
   ///   lower bounds of the `bounds` range must be valid indices of the
   ///   collection.
+  ///
+  /// - Complexity: O(1)
```

Comment:
Is this agreed/inferred from somewhere else? Wonder if there are implementations where this is desirably not the case.